lms/djangoapps/static_template_view/urls.py

```diff
@@ -18,6 +18,7 @@
     url(r'^contact$', views.render, {'template': 'contact.html'}, name="contact"),
     url(r'^donate$', views.render, {'template': 'donate.html'}, name="donate"),
     url(r'^faq$', views.render, {'template': 'faq.html'}, name="faq"),
+    url(r'^what_is_soa', views.render, {'template': 'what_is_soa.html'}, name="what_is_soa"),
```

Comment:
```suggestion
    url(r'^what_is_soa$', views.render, {'template': 'what_is_soa.html'}, name="what_is_soa"),
```
i think it would be better this way.

Reply:
done.